packages/fluentui/react-northstar/src/themes/teams/components/Divider/dividerStyles.ts

```diff
@@ -40,11 +40,13 @@ const dividerStyles: ComponentSlotStylesPrepared<DividerStylesProps, DividerVari
             lineHeight: variables.textLineHeight,
             '::before': {
               ...beforeAndAfter(size, variables, colors, props),
-              marginRight: pxToRem(20),
+              ...(!rtl && { marginRight: pxToRem(20) }),
+              marginInlineEnd: pxToRem(20),
```

Comment:
`margin-inline-end` is [not supported in IE11](https://caniuse.com/#search=margin-inline-end).

Reply:
The alternative could be create a slot for the text and apply the margins on it, but @layershifter mentioned it can be too much to solve this...

Initially I didn't saw a problem in the related issue itself, the RTL text works in RTL context and LTR in the LTR... But if we are going to solve it what can be the best solution then?

Reply:
I pushed new code solving it with a slot.

Reply:
You are right @assuncaocharles, initially it looks like there is no an issue (when the direction of the language meets the direction of the context flow), but in some cases there may be a need to fill that content with a language that has a different direction than the context flow direction. 
BTW, I've mentioned the main reason that leads to this issue, [here](https://github.com/microsoft/fluentui/issues/13337#issuecomment-634954807).

In other words:
> The `auto` value tells the browser to look at the first strongly typed character in the element and work out from that what the base direction of the element should be.
[w3.org](https://www.w3.org/International/questions/qa-html-dir#dirauto)

Therefore, I would suggest removing that `auto` if there is no any other side effect should be considered.